src/store/events.store.js

```diff
@@ -0,0 +1,32 @@
+import db from '../assets/db.json';
+
+export default {
+	state: {
+		events: db.events,
+	},
+	getters: {
+		getEventsList: (state) => state.events,
```

Comment:
It's a convention to use nouns for getters/computed property instead of verbs - `eventsList`.

Reply:
Getters renamed